src/bin/lttng/utils.c

```diff
@@ -387,7 +387,7 @@ int check_relayd(void)
 	 * A successful connect means the relayd exists thus returning 0 else a
 	 * negative value means it does NOT exists.
 	 */
-	ret = connect(fd, &sin, sizeof(sin));
+	ret = connect(fd, (void *) &sin, sizeof(sin));
```

Comment:
Which libc needs "void *" here?

Reply:
Regardless of the chosen libc, `connect(2)` seems to accept a `const struct sockaddr` pointer and we're passing a `struct sockaddr_in` pointer. This is an illegal conversion between pointer types AFAIK.